simulator/opendc-experiments/opendc-experiments-allocateam/src/main/kotlin/org/opendc/experiments/allocateam/experiment/monitor/RunMonitor.kt

```diff
@@ -59,19 +66,26 @@ public class RunMonitor(private val run: Run, private val clock: Clock) {
     }
 
     public fun reportJobStarted(event: WorkflowEvent.JobStarted) {
-        startTimesPerJob[event.job] = event.time
+        startTimesPerJob[event.jobState.job] = event.time
         this.startedJobs += 1
+        submissionTimesPerJob[event.jobState.job] = event.jobState.submittedAt
```

Comment:
Should the submission time be updated when a job is started?

Reply:
I know that it's not the proper way of doing it, but otherwise, we'd have to add another listener for when a job has been submitted and I don't think we'll need it, so I just did it this way, it doesn't change anything tbh.